src/main/java/org/springframework/data/tarantool/core/query/TarantoolRepositoryObjectQueryExecutor.java

```diff
@@ -7,13 +7,14 @@
  * annotation and having result as object or list of objects
  *
  * @author Oleg Kuzentsov
+ * @author Artyom Dubinin
  */
-public class TarantoolRepositoryQueryObjectExecutor implements TarantoolRepositoryExecutor {
+public class TarantoolRepositoryObjectQueryExecutor implements TarantoolRepositoryExecutor {
```

Comment:
See other comments about renaming. `RepositoryQuery` is a Spring Data class, we need to keep track of it.

Reply:
renamed, now:
TarantoolTupleRepositoryQuery
TarantoolTupleRepositoryQueryExecutor
TarantoolObjectRepositoryQuery
TarantoolObjectRepositoryQueryExecutor